src/components/PlantPhotoSwiper/PlantPhotoSwiper.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Dimensions } from 'react-native';
+import { View } from 'react-native';
+import Swiper from 'react-native-swiper';
+import FastImage from 'react-native-fast-image';
+import { useStyles } from 'react-native-unistyles';
+
+import { styleSheet } from './PlantPhotoSwiper.style';
+import { PlantPhoto } from '@src/api/model/plantPhoto';
+import { commonColors } from '@src/styling/commonColors';
+
+type PlantPhotoSwiperProps = {
+	plantPhotos: PlantPhoto[];
+};
+
+export const PlantPhotoSwiper = ({ plantPhotos }: PlantPhotoSwiperProps) => {
+	const { styles } = useStyles(styleSheet);
+
+	const width = Dimensions.get('window').width;
+
+	return (
+		<View style={styles.wrapper}>
+			<Swiper loop activeDotColor={commonColors.white} width={width}>
+				{plantPhotos.map((photo) => (
+					<FastImage
+						key={photo?.urlPath}
```

Comment:
How can photo be optional here?

Reply:
If we don't have photos, the photo swiper component will not be shown at all

Reply:
Yes, then why do we have ? Near each photo prop?